apps/etterlatte-behandling/src/main/resources/db/migration/V49__lag_oppgaver_for_behandlinger_og_hendelser.sql

```diff
@@ -0,0 +1,137 @@
+                                     case source.behandlingstype
+                                         when 'FØRSTEGANGSBEHANDLING' then 'FOERSTEGANGSBEHANDLING'
+                                         else source.behandlingstype
+                                         end
+                                 else 'UNDERKJENT'
+                                 end
+                         when 'VEDTAK:UNDERKJENT' then 'ATTESTERING'
+                         when 'VEDTAK:ATTESTERT' then 'ATTESTERING'
+                         when 'BEHANDLING:AVBRUTT' then
+                             case source.rekkefolge
+                                 when 1 then
+                                     case source.behandlingstype
+                                         when 'FØRSTEGANGSBEHANDLING' then 'FOERSTEGANGSBEHANDLING'
+                                         else source.behandlingstype
+                                         end
+                                 else case (SELECT hendelse
+                                            from source s1
+                                            where source.behandlingid = s1.behandlingid
+                                              and s1.motsatt_rekkefolge = 2)
+                                          when 'VEDTAK:UNDERKJENT' then 'UNDERKJENT'
+                                          when 'VEDTAK:ATTESTERT' then 'SKAL_IKKE_SKJE'
+                                          when 'VEDTAK:FATTET' then 'ATTESTERING'
+                                     end
+                                 end
+                         end                                          as type,
+                     source.saktype                                   as saktype,
+                     source.fnr                                       as fnr,
+                     source.behandling_opprettet + interval '1 month' as frist,
+                     'BEHANDLING'                                     as kilde
+              FROM source);
+
+-- få med en åpen oppgave av riktig type for alle behandlinger som ikke er avsluttet. typen oppgave kan utledes
+-- fra den siste behandlingshendelsen som skjedde på denne behandlingen.
+-- Siden alle behandlinger får hendelsen "'BEHANDLING:OPPRETTET'" med en gang kan vi garantere at vi har
+-- minst en hendelse per behandling
+WITH hendelser_med_rekkefolge as (select *,
+                                         row_number() over (partition by behandlingid order by opprettet desc) as motsatt_rekkefolge
+                                  from behandlinghendelse hendelser
+                                  order by opprettet asc),
+     siste_hendelse_per_behandling
+         as (select * from hendelser_med_rekkefolge where motsatt_rekkefolge = 1) -- motsatt_rekkefolge = 1 <=> siste hendelse
+INSERT
+INTO oppgave (SELECT gen_random_uuid()                                    as id,
+                     'UNDER_ARBEID'                                       as status,
```

Comment:
Denne skal vel være status=`NY`.

Reply:
you're goddamn right

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/1867/commits/5de378ddf8f1f5f8eeddcb8e0f272ab42e182f0e